README.md

```diff
@@ -227,6 +227,10 @@ TODO: Move this code over to the idseq-dag repo.
 
 ## Release notes
 
+- 3.13.1 - 3.13.2
+  - Stability improvements with s3 concurrency, retries, and random deadlocks.
+  - Bugfix in reference caching (GITHUB PR 218, JIRA IDSEQ-1759)
```

Comment:
The "bugfix in reference caching" went in prior to this without a version update.  Just catching up with the versions here.  It's not a part of this PR though.